pycqed/measurement/waveform_control/circuit_builder.py

```diff
@@ -33,6 +34,7 @@ def __init__(self, dev=None, qubits=None, operation_dict=None,
             dev, qubits, operation_dict, filter_qb_names)
         self.update_operation_dict(operation_dict)
         self.cz_pulse_name = kw.get('cz_pulse_name', 'upCZ')
+        self.decompose_rotation_gates = kw.get('decompose_rotation_gates', {})
```

Comment:
new kw not explained (?). Does this refer to a decomposition using several single qubit gates?